src/freertos_drivers/st/Stm32Can.hxx

```diff
@@ -91,11 +97,17 @@ private:
     static unsigned int intCount;
 
     uint8_t state_; ///< present bus state
-
+    
+    CAN_TypeDef *can_;
```

Comment:
Please add Doxygen comments for these members.

Reply:
Done.